src/api/controllers/members.ts

```diff
@@ -62,13 +62,14 @@ export async function handleInternalGuildMemberUpdate(data: DiscordPayload) {
     deaf: guildMember?.deaf || false,
     mute: guildMember?.mute || false,
     roles: payload.roles,
+    nick: payload.nick === undefined ? null : payload.nick,
```

Comment:
Same here, this PR should solely consists of TypeScript types' changes.

Reply:
Can add `!` here too